easybuild/easyconfigs/m/MUMPS/MUMPS-5.4.1-foss-2021b-metis.eb

```
@@ -0,0 +1,35 @@
+name = 'MUMPS'
+version = '5.4.1'
+versionsuffix = '-metis'
+
+homepage = 'https://graal.ens-lyon.fr/MUMPS/'
+description = """A parallel sparse direct solver"""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['http://mumps.enseeiht.fr/']
+sources = ['%(name)s_%(version)s.tar.gz']
+patches = [
+    '%(name)s-%(version)s_shared-pord.patch',    # builds the shared libs of PORD
+    '%(name)s-%(version)s_shared-mumps.patch',   # builds shared libs of MUMPS
+]
```

since the patch files haven't changed since `5.4.0`, we can just use those:
```suggestion
patches = [
    '%(name)s-5.4.0_shared-pord.patch',    # builds the shared libs of PORD
    '%(name)s-5.4.0_shared-mumps.patch',   # builds shared libs of MUMPS
]
```